pyvista/__init__.py

```diff
@@ -74,3 +54,45 @@
 
 # Name used for unnamed scalars
 DEFAULT_SCALARS_NAME = 'Data'
+
+MAX_N_COLOR_BARS = 10
+
+
+# Lazily import/access the plotting module
+def __getattr__(name):
+    """Fetch an attribute ``name`` from ``globals()``or the ``pyvista.plotting`` module..
+
+    This override is implemented to prevent importing all of the plotting module
+    and GL-dependent VTK modules when importing PyVista.
+
+    Raises
+    ------
+    AttributeError
+        If the attribute is not found.
+
+    """
+    try:
+        return globals()[name]
+    except KeyError:
+        pass
+
+    import importlib
+    import inspect
+
+    whitelist = [
```

Comment:
Mild recommendation to go with the times and call this `allow_list` or `shortlist` (or something else) instead. And make it a set for slightly neater lookup?